src/ChildProcess.php

```diff
@@ -7,19 +7,19 @@
 
 class ChildProcess implements ChildProcessContract
 {
-    public readonly int $pid;
+    public int $pid;
 
-    public readonly string $alias;
+    public string $alias;
 
-    public readonly array $cmd;
+    public array $cmd;
 
-    public readonly ?string $cwd;
+    public ?string $cwd;
 
-    public readonly ?array $env;
+    public ?array $env;
 
-    public readonly bool $persistent;
+    public bool $persistent;
```

Comment:
These changes feel like a regression. Readonly here ensures these values don't get changed. They shouldn't change as they represent immutable state rather than being a mutable object.

Reply:
PHPStan informed me that we cannot update a readonly property outside of its constructor. I initially thought this might be an issue in PHP 8.4, but I just tested it in both PHP 8.3 and 8.4, and we can assign a value without any problems.

I reviewed the documentation, and it does not state that assignments outside of the constructor are prohibited. A readonly property simply needs to be initialized once from within the class.

According to the [documentation](https://www.php.net/manual/en/language.oop5.properties.php)
> A readonly property can only be initialized once, and only from the scope where it has been declared. Any other assignment or modification of the property will result in an Error exception.

So..


→ I can revert this commit and add a phpstanignore tag instead! Agree?

Reply:
Yeh, I think let's bring `readonly` back here... I'm not sure PHPStan is on about